verify.rb

```diff
@@ -54,10 +58,11 @@ def test_img_file(img)
   end
 
   # Check image permissions
-  perms = File.stat(img).mode
-  error("#{img} is not set 644. It is currently #{perms.to_s(8)}")\
-    unless perms.to_s(8) == '100644'
+  perms = File.stat(img).mode.to_s(8).split(//).last(3).join
+  error("#{img} is not set 644 or 664. It is currently #{perms}")\
```

Comment:
For future-proofing, this should be changed to use the new instance variable's values:
`error("#{img} permissions must be one of: #{@img_permissions.join(',')}. It is currently #{perms}") `